multitenancy/tenant-manager-datamodel/src/main/java/io/apicurio/multitenant/api/datamodel/RegistryTenant.java

```diff
@@ -38,6 +41,7 @@
      * (Required)
      * 
      */
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
     @JsonProperty("createdOn")
     @JsonPropertyDescription("Date when the tenant was created")
```

Comment:
I'm using `RFC3339 in the UTF timezone` in descriptions of date/time fields in the Fleet manager, which is basically the same I think, but may be more helpful for clients to assume a standard.